### PR TITLE
Removing unused versions sdk

### LINKED ISSUE #483: Update Android Gradle Plugin
We have a rather old version of Gradle & Android Gradle Plugin running, and it makes it harder to work on our configuration like in the case of #379 . It would also fix the warnings, as noted in #336 

Update Android Gradle Plugin from 3.6.4 to version 7.2.2.

### LINKED ISSUE #484: Remove or Upgrade android-sdk-versions-plugin to latest version
The android-sdk-versions-plugin is currently blocking an Upgrade of Grade #483 .

The mapbox/android-sdk-versions-plugin [repo](https://github.com/mapbox/android-sdk-versions-plugin) isn't publishing the latest changes to [maven central](https://search.maven.org/artifact/com.mapbox.mapboxsdk/mapbox-android-sdk-versions/1.1.0/jar), with the latest published version being 1.1.0, and the latest repo version 1.1.3.

The version we need is >= 1.1.1.

We can publish it in our Nexus account and move to it instead. 

Work on cloning and publishing it can be found in here: https://github.com/birkskyum/android-sdk-versions-plugin

### PR DESCRIPTION
**Pull request by birkskyum**
Fixes #484 by removing a problematic [package](https://github.com/mapbox/android-sdk-versions-plugin) that was used for telemetry in the past but is not in use anymore.

### CODE REVIEW

File: .github/workflows/android-ci-pull.yml [hunk 1]
```diff
@@ -59,6 +59,9 @@ jobs:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Setup Android SDK
```
Comment (ntadej) [inline .github/workflows/android-ci-pull.yml:62]: Why is this needed? The latest SDK should already be set in GitHub actions.

**Review by ntadej — DISMISSED** [review]

File: .github/workflows/android-ci-pull.yml (hunk 1 above)
Comment (birkskyum) [reply .github/workflows/android-ci-pull.yml:62]: https://github.com/maplibre/maplibre-gl-native/actions/runs/3142254689/jobs/5105640527

File: .github/workflows/android-ci-pull.yml [hunk 2]
```diff
@@ -59,6 +59,14 @@ jobs:
...
+        uses: android-actions/setup-android@v2
+
+      - name: Setup cmake
```
[the remark this answers is not in the document]
Comment (birkskyum) [reply .github/workflows/android-ci-pull.yml:65]: because its the one required by the android docker container from platforms/android/docker that it all runs on
https://github.com/maplibre/maplibre-gl-native/actions/runs/3142446133/jobs/5106047299

File: .github/workflows/android-ci-pull.yml (hunk 1 above)
Comment (birkskyum) [reply .github/workflows/android-ci-pull.yml:62]: it says the license isn't accepted otherwise

**Comment by birkskyum** [comment]
I've reduced the scope of the PR to not include the bump of Gradle.

**Review by carlonzo — APPROVED** [review]
I guess that's it! Again i can't find any reference to it, let's see
